source/util.js

```diff
@@ -61,3 +64,27 @@ exports.getTagVersionPrefix = pMemoize(async options => {
 		return 'v';
 	}
 });
+
+exports.checkIgnoreStrategy = ({files}) => {
+	const npmignoreExists = existsSync(path.resolve(getRootDir(), '.npmignore'));
+
+	if (!files && !npmignoreExists) {
+		console.log(
+			`\n
+╭───────────────────────────────────────────────────╮
+│                                                   │
+│                 ⚠️  Warning ⚠️                      │
+│   No ${chalk.bold.magenta('files')} field specified in ${chalk.bold.magenta('package.json')}        │
```

Comment:
```suggestion
│   No ${chalk.bold.cyan('`files`')} field specified in ${chalk.bold.magenta('`package.json`')}        │
```

To better distinguish property names and filenames (feel free to use any other color in case you think it's a better fit).

Reply:
done